File: py/1168.py
numbers = [6, 2, 5, 5, 4, 5, 6, 3, 7, 6]

total = 0
result = []
count = int(input())

for i in range(count):
    case = input()
    
    for n in case: total += numbers[int(n)]
        
    result.append(total)
    total = 0
        
for i in result: print(f'{i} leds')

File: cpp/1168.cpp
#include <iostream>
#include <string>

using namespace std;

int main()
{
    int numbers[10] = {6, 2, 5, 5, 4, 5, 6, 3, 7, 6}, total = 0, n, len, i;
    cin >> n;

    for (n; n > 0; n--) {
        string number;
        cin >> number;
        len = number.length();

        for (i = 0; i < len; i++) {
            total += numbers[ (int)(number[i]) - 48 ];
        }

        cout << total << " leds" << '\n';
        total = 0;
    }

    return 0;
}